mne_sandbox/preprocessing/bads/bads.py

```diff
@@ -0,0 +1,263 @@
+
+    Parameters
+    ----------
+    epochs : Instance of Epochs
+        The epochs for which bad channels need to be marked
+    picks : list of int | None
+        Channels to operate on. Defaults to EEG channels.
+    method : {'faster'}
+        The detection algorithm.
+    method_params : dict | None
+        The method parameters in a dict.
+
+        If ``method`` equals 'faster', and ``method_params`` is None,
+        defaults to the following parameters. Partial updates are supported.
+        use_metrics : list of str
+            List of metrics to use. Can be any combination of:
+                'variance', 'correlation', 'hurst', 'kurtosis', 'line_noise'
+            Defaults to all of them.
+        thresh : float
+            The threshold value, in standard deviations, to apply. A channel
+            crossing this threshold value is marked as bad. Defaults to 3.
+        max_iter : int
+            The maximum number of iterations performed during outlier detection
+            (defaults to 1, as in the original FASTER paper).
+        eeg_ref_corr : bool
+            If the EEG data has been referenced using a single electrode
+            setting this parameter to True will enable a correction factor for
+            the distance of each electrode to the reference. If an average
+            reference is applied, or the mean of multiple reference electrodes,
+            set this parameter to False. Defaults to False, which disables the
+            correction.
+
+    return_by_metric : bool
+        Whether to return the bad channels as a flat list (False, default) or
+        as a dictionary with the names of the used metrics as keys and the
+        bad channels found by this metric as values. Is ignored if not
+        supported by method.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+        Defaults to self.verbose.
+
+    Returns
+    -------
+    bads : list of str
+        The names of the bad EEG channels.
+
+    See Also
+    --------
+    find_bad_epochs
+    find_bad_channels_in_epochs
+
+    References
+    ----------
+    [1] H., Whelan R. and Reilly RB. FASTER: fully automated statistical
+    thresholding for EEG artifact rejection. Journal of Neuroscience Methods,
+    vol. 192, issue 1, pp. 152-162, 2010.
+    """
+    if picks is None:
+        picks = pick_types(epochs.info, meg=True, eeg=True, exclude=[])
+    _method_params = _handle_default('bad_channels' + '_' + method,
+                                     method_params)
+    if method == 'faster':
+        bads = _faster._find_bad_channels(epochs, picks, **_method_params)
+    else:
+        raise NotImplementedError(
+            'Come back later, for now there is only "FASTER"')
+
+    if return_by_metric:
+        return bads
+    else:
+        return _combine_indices(bads)
+
+
+@verbose
+def find_bad_epochs(epochs, picks=None, return_by_metric=False,
```

Comment:
I would actually make `method` the second argument since it's probably more important than `picks`